website/blog/2023-3-01-fonicom.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: fonicom
+title: "How Corso helps Fonicom deliver a more reliable experience for internal backups"
+description: "Corso, an open-source tool for backing up Microsoft 365 Data, was lucky to have Kias Hanifa from Fonicom take a look at our tool. He was kind enough to write this guest post."
+authors: kias
+tags: [corso, microsoft 365, backups, MSP]
+date: 2023-3-01
+image: ./images/servers.png
+---
+
+![media server By Victor Grigas/Wikimedia Foundation, CC BY-SA 3.0, https://commons.wikimedia.org/w/index.php?curid=44043618](./images/servers.png)
+
+*Corso, an open-source tool for backing up Microsoft 365 Data, was lucky to have
```

Comment:
This is broken when compiled. Please remember to preview blog posts before requesting reviews.

Reply:
will preview visually in future

Reply:
fixed